grpc_tensorflow_server/grpc_tensorflow_server.py

```diff
@@ -99,9 +99,16 @@ def main(_):
   if not FLAGS.job_name:
     raise ValueError("Empty job_name")
   server_def.job_name = FLAGS.job_name
-
+  job_def = None
+  for i in range(len(server_def.cluster.job)):
+    if server_def.cluster.job[i].name == FLAGS.job_name:
+      job_def = server_def.cluster.job[i]
+      break
+  if job_def is None:
```

Comment:
can use the infamous `for in ... else` loop

Reply:
```for in ... else``` doesn't seem to be very common in Python code. I prefer the current, verbose implementation since that will be more familiar to Python coders.